client/src/components/Projects/ProjectEdit/index.jsx

```diff
@@ -30,8 +30,10 @@ export default class index extends Component {
           data: { data },
           status
         } = result;
-        console.log(data);
         if (status === 200) {
+          if (!data.description) {
+            data.description = " ";
+          }
```

Comment:
@aminalakhsham 👍 
If you want, you could put this into the render function when you are displaying description (then you aren't changing the internal data, so you could re-save that same data without changing undefined descriptions into empty strings if that matters)